### DISCUSSION TITLE
Select - How Best to Remove SelectArrow

### DISCUSSION THREAD

**Discussion by alexiscordova** [ariakit/ariakit#2992] [Questions] [answered]
Hi there. I'm attempting to pass a custom component to `<Select>` through the render method, but when I use an Input I get the following error:

```
input is a void element tag and must neither have `children` nor use `dangerouslySetInnerHTML`.
```

This seems to occur because the Select component is automatically adding the arrow SVG as a child of the custom component, and Inputs cannot have child nodes. Is it possible to remove the SelectArrow so I can use an input as my trigger component? Thanks!

**Comment by diegohaz** [comment, marked as answer]
If you're trying to render [Select](https://ariakit.org/components/select) as an input, you probably want a [Combobox](https://ariakit.org/components/combobox), not Select.

But you can omit the `children` prop using the [`render` function](https://ariakit.org/guide/composition#explicit-render-function):

```jsx
<Select
  render={({ children, ...props }) => <input {...props} />}
/>
```

**Reply by alexiscordova** [reply]
Ah of course! Thanks for the quick reply
